src/interfaces/records/handlers/records-write.ts

```diff
@@ -120,6 +121,9 @@ export async function constructRecordsWriteIndexes(
     recordId  : message.recordId,
     entryId   : await RecordsWrite.getEntryId(recordsWrite.author, recordsWrite.message.descriptor)
   };
+  if (descriptor.protocol !== undefined) {
```

Comment:
NIT: rather than check against the `descriptor.protocol`, should we check `indexes.protocol` since ultimately we really only care about normalizing the `protocol` value in the `IndexLevel`?  some part of me feels like it's odd to have it be added to `indexes` only to then check some other place before overriding it